app/controllers/studios_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StudiosController < ApplicationController
+  def index
+    @studios = Studio.all
+  end
+
+  def show
+    @studio = Studio.find(params[:id])
+    @actors = @studio.actors.uniq
```

Comment:
This is the right idea in that it returns us non-repeating actors, but `.uniq` is a Ruby method, and we want to lean on AR as much as possible. Instead, let's use `.distinct`. Try that in a Terminal in `rails c` and you'll see the query turns into: 

```sql
SELECT DISTINCT ...
```
..which means we're altering the query, and since we're in a controller, we should move this to a Model method instead.